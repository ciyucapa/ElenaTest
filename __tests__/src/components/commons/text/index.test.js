import React from 'react';
import {create} from 'react-test-renderer';

import Text from '../../../../../src/components/commons/text';

describe('Text component', () => {
    test('render text component', () => {
        const tree = create(
            <Text text={'Text'} />,
        ).toJSON();
        expect(tree).toMatchSnapshot();
    });
});