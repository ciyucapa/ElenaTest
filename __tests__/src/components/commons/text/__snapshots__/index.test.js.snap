// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text component render text component 1`] = `
<div>
  Text
</div>
`;
